klippy/kinematics/extruder.py

```diff
@@ -72,6 +72,12 @@ def __init__(self, config, extruder_num):
         gcode.register_mux_command("ACTIVATE_EXTRUDER", "EXTRUDER",
                                    self.name, self.cmd_ACTIVATE_EXTRUDER,
                                    desc=self.cmd_ACTIVATE_EXTRUDER_help)
+        gcode.register_mux_command("GET_EXTRUDER_STEP_DISTANCE", "EXTRUDER",
+                                   self.name, self.cmd_GET_E_STEP_DISTANCE,
+                                   desc=self.cmd_GET_E_STEP_DISTANCE_help)
```

Comment:
FWIW, I would not add a GET_EXTRUDER_STEP_DISTANCE command.  Instead, I'd have SET_EXTRUDER_STEP_DISTANCE report the current step distance if an explicit DISTANCE parameter is not specified.

Reply:
Makes sense I will make that change.